tests/DataFixtures/ORM/resources/cart.yml

```diff
@@ -0,0 +1,87 @@
+Sylius\Component\Core\Model\Order:
+    order_001:
```

Comment:
It just the way how we define order fixtures with alice. I just wanted to be consistent. It doesn't change anything. 🐋